### ISSUE TITLE
Using Generic OAuth provider results in no button

### ISSUE THREAD

**Issue by chauffer** [Requarks/wiki#383]
### Actual behavior
![](https://simone.sh/images/01250003_ghule.jpg)

### Expected behavior
![](https://simone.sh/images/01250003_pdipm.jpg)

### Steps to reproduce the behavior
```
public: false
auth:
  defaultReadAccess: false
  local:
    enabled: false
  oauth2:
    enabled: true
    clientId: xxx
    clientSecret: xxx
    authorizationURL: https://xxx
    tokenURL: https://xxx
```

**Comment by NGPixel** [comment]
Fixed in 1.0.76

**Comment by danvim** [comment]
@NGPixel Sorry, I can't seem to find logging in with a generic OAuth provider in docs. Is it in there or did I miss it?
